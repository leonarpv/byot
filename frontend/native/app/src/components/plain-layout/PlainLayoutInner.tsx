import * as React from 'react';
import {StyleSheet, View} from 'react-native';

interface Props {
  children?: React.ReactNode;
}

interface State {}

const makeStyles = (props: Props, state: State) =>
  StyleSheet.create({
    root: {
      flex: 1,
      justifyContent: 'flex-end',
      flexDirection: 'column',
    },
  });

export const PlainLayoutInner: React.FC<Props> = (props: Props) => {
  const styles = makeStyles(props, {});
  return <View style={styles.root}>{props.children}</View>;
};
