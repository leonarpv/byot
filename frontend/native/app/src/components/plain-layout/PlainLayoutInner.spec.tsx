import * as React from 'react';
import {render} from '@testing-library/react-native';
import {PlainLayoutInner} from './PlainLayoutInner';

describe('<PlainLayoutInner/>', () => {
  it('should render', () => {
    const {asJSON} = render(<PlainLayoutInner />);
    expect(asJSON()).toMatchSnapshot();
  });
});
