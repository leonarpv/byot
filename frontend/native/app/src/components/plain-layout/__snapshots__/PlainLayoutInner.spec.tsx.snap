// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlainLayoutInner/> should render 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "flex-end",
      }
    }
  />
</View>
`;
